app/Party.php

```diff
@@ -1055,10 +1055,14 @@ public function approve()
         }
 
         // Notify the person who created it that it has now been approved.
-        $host = User::find(EventsUsers::where('event', $this->idevents)->first());
+        $eu = EventsUsers::where('event', $this->idevents)->first();
```

Comment:
Are we certain this is always the person who created the event?  

I guess it is relying on:

- the person who created the event always having the lowest id in events_users
- the query always returning ordered by events users id ascending

I think the first one is fair for now, though we might be better off having an explicit 'creator' flag of some kind when we revisit events_users statuses.

The second one - no clue, would be good to have a quick check in Laravel docs.

Reply:
I've added an orderBy which will do until we add a creator.